src/main/java/message/Request.java

```diff
@@ -0,0 +1,82 @@
+package main.java.message;
+
+
+import org.apache.commons.lang3.StringUtils;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Request implements Serializable {
+    private String method;
+    private String path;
+    private String version;
+    private Map<String, String> headers;
+    private String contents;
+
+    public static Request init(BufferedReader br) throws IOException {
+        Request request = new Request();
+        request.makeStartLine(br.readLine())
+                .makeHeader(br);
+        return request;
+
+    }
+
+    private Request makeStartLine(String startLine) {
+        if(startLine == null){
+            new IllegalArgumentException("first line is null. It is invalid data format.");
+        }
+
+        String[] array = startLine.split(" ");
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: A "NullPointerException" could be thrown; "startLine" is nullable here. ([squid:S2259](https://sonarcloud.io/organizations/jangchulwoon-github/rules#rule_key=squid%3AS2259))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=groupId%3AHTTPServer&pullRequest=4&issues=AWQvx9PN_bEzQ-kUjrAr&open=AWQvx9PN_bEzQ-kUjrAr)

[ISSUE_KEY]: AWQvx9PN_bEzQ-kUjrAr